cli/command/image/trust.go

```diff
@@ -301,8 +299,11 @@ func trustedPull(ctx context.Context, cli *command.DockerCli, repoInfo *registry
 		}
 
 		// If reference is not trusted, tag by trusted reference
-		if !r.reference.HasDigest() {
-			tagged, err := reference.WithTag(repoInfo, r.reference.String())
+		// TODO(aaronl): Is this code really necessary? Will notary ever
```

Comment:
> Are you asking whether the check is necessary since Notary is always mapping untrusted tags to digest?

Yes

> This block should always end up getting called.

Are you saying it's safe to remove the check on whether the target name is a digest? My guess was that it should be removed, but I wanted to confirm before making a change like this.